pipeline/ffmpeg.go

```diff
@@ -15,11 +16,21 @@ import (
 	"github.com/livepeer/catalyst-api/video"
 )
 
+const LocalSourceFilePattern = "sourcevideo*"
```

Comment:
I changed the pattern here from `source*` to `sourcevideo*` just to try and reduce the chance of catching some other unrelated file that's in the tmp directory

Reply:
This does mean that following the deploy where this goes out, we should probably do manual cleanup of any files with the old `source*` pattern